Specs/Scene/ModelExperimental/CustomShaderPipelineStageSpec.js

```diff
@@ -263,33 +217,26 @@ describe("Scene/ModelExperimental/CustomShaderPipelineStage", function () {
   });
 
   it("generates shader code from built-in attributes", function () {
-    const shaderBuilder = new ShaderBuilder();
-    const model = {
-      customShader: new CustomShader({
-        vertexShaderText: [
-          "void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput)",
-          "{",
-          "    vec3 normalMC = vsInput.attributes.normalMC;",
-          "    vec2 texCoord = vsInput.attributes.texCoord_0;",
-          "    vsOutput.positionMC = vsInput.attributes.positionMC;",
-          "}",
-        ].join("\n"),
-        fragmentShaderText: [
-          "void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)",
-          "{",
-          "    vec3 positionMC = fsInput.attributes.positionMC;",
-          "    vec3 normalEC = fsInput.attributes.normalEC;",
-          "    vec2 texCoord = fsInput.attributes.texCoord_0;",
-          "}",
-        ].join("\n"),
-      }),
-    };
-    const renderResources = {
-      shaderBuilder: shaderBuilder,
-      model: model,
-      lightingOptions: new ModelLightingOptions(),
-      alphaOptions: new ModelAlphaOptions(),
-    };
+    const customShader = new CustomShader({
+      vertexShaderText: [
+        "void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput)",
+        "{",
+        "    vec3 normalMC = vsInput.attributes.normalMC;",
+        "    vec2 texCoord = vsInput.attributes.texCoord_0;",
+        "    vsOutput.positionMC = vsInput.attributes.positionMC;",
+        "}",
+      ].join("\n"),
+      fragmentShaderText: [
+        "void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)",
+        "{",
+        "    vec3 positionMC = fsInput.attributes.positionMC;",
+        "    vec3 normalEC = fsInput.attributes.normalEC;",
+        "    vec2 texCoord = fsInput.attributes.texCoord_0;",
+        "}",
+      ].join("\n"),
+    });
```

Comment:
Oh I see that this code still uses an array of strings and joining the results. We can use ES6 template literals, which do support multi-line strings. For example:

```
    const customShader = new CustomShader({
      vertexShaderText: `
        void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput)
        {
            vec3 normalMC = vsInput.attributes.normalMC;
            vec2 texCoord = vsInput.attributes.texCoord_0;
            vsOutput.positionMC = vsInput.attributes.positionMC;
        }
      `,
      ...
```